src/test/java/edu/harvard/iq/dataverse/search/IndexServiceBeanTest.java

```diff
@@ -51,6 +51,7 @@ public void setUp() {
         indexService.settingsService = Mockito.mock(SettingsServiceBean.class);
         indexService.dataverseService = Mockito.mock(DataverseServiceBean.class);
         indexService.datasetFieldService = Mockito.mock(DatasetFieldServiceBean.class);
+        indexService.datasetVersionFilesServiceBean = Mockito.mock(DatasetVersionFilesServiceBean.class);
         BrandingUtil.injectServices(indexService.dataverseService, indexService.settingsService);
 
         Mockito.when(indexService.dataverseService.findRootDataverse()).thenReturn(dataverse);
```

Comment:
Is it worth adding a test to SearchIT.java?

Reply:
- As IT is to test output of API call, I would say no nothing change here and there is already "something" https://github.com/IQSS/dataverse/blob/develop/src/test/java/edu/harvard/iq/dataverse/api/SearchIT.java#L433

- I Tried to add a test inside `IndexServiceBeanTest.java` but as we play with mocked dataset, datafile, metadatafile and mocked DatasetVersionFilesServiceBean I ended up with `fileCount=0` So I don't see a good way to add a test.